lib/storage.js

```diff
@@ -441,6 +441,42 @@ exports.set = function(key, json, options, callback, retries) {
   });
 };
 
+/**
+ * @summary Write user data sync
+ * @function
+ * @public
+ *
+ * @param {String} key - key
+ * @param {Object} json - json object
+ * @param {Function} callback - callback (error)
+ *
+ * @example
+ * const storage = require('electron-json-storage');
+ *
+ * storage.setSync('foobar', { foo: 'bar' }, function(error) {
+ *   if (error) throw error;
+ * });
+ */
+exports.setSync = function(key, json, options, callback) {
+    const fileName = utils.getFileName(key, {
```

Comment:
I think indentation is off here. It should be two spaces based on the other contents of this file

Reply:
Done